public/kbn_sankey_vis_controller.js

```diff
@@ -10,20 +11,24 @@ import { filterNodesAndLinks } from './lib/filter';
 const module = uiModules.get('kibana/kbn_sankey_vis', ['kibana']);
 let observeResize = require('./lib/observe_resize');
 
+const IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
+
 module.controller('KbnSankeyVisController', function ($scope, $element, $rootScope, Private) {
   const sankeyAggResponse = Private(AggResponseProvider);
   $scope.emptyGraph = false;
 
   let svgRoot = $element[0];
   let resize = false;
-  let color = d3.scale.category10();
+  let color = d3.scale.category20();
   let margin = 20;
   let width;
   let height;
   let div;
   let svg;
   let globalData = null;
 
+  let textColor = (IS_DARK_THEME) ? '#CBCFCB' : '#000000';
```

Comment:
Hi @mmguero , I think it will be better if the different colors are defined separately in another file, something like this:
`import * as colors from './lib/colors';`
`....`
`let textColor = (IS_DARK_THEME) ? colors.DARK_COLOR : colors.LIGHT_COLOR;`

Reply:
I agree, I think that would be a good idea. I will make that change and update the pull request. I actually found [a better-supported way](https://discuss.elastic.co/t/detecting-dark-theme-in-visualization-plugin/235220/3) to do it, so I will tweak that as well.